test/test_numerizer_en.rb

```diff
@@ -24,7 +31,7 @@ def test_straight_parsing
       59 => 'fifty nine',
       100 => ['one hundred', 'a hundred', 'hundred a'],
       150 => 'one hundred and fifty',
-      # 150 => 'one fifty',
+      150 => 'one fifty',
```

Comment:
just noticed that this uses the same key as the one above so only one is kept in the hash, you need to put the two two cases into an array like we have for the `100` case.